src/platform_impl/windows/event_loop.rs

```diff
@@ -1095,6 +1098,104 @@ unsafe fn public_window_callback_inner<T: 'static>(
             0
         }
 
+        WM_IME_STARTCOMPOSITION => {
+            let ime_allowed = { userdata.window_state.lock().ime_allowed };
```

Comment:
Hm, I don't think you need to brackets around, but it could probably be `*userdata.window_state.lock().ime_allowed`. I understand that you want to release lock right away.

Reply:
done, used `userdata.window_state.lock().ime_allowed`.